add non-x86_64 architecture libfuzzer target support using qemu-user (#600)
microsoft/onefuzz: docs/how-to/fuzzing-other-architectures-on-azure.md

# Fuzzing non-X86_64 targets on Azure

Fuzzing non-x86_64 targets using libFuzzer on Azure makes use of two Open
Source capabilities:

2. [libFuzzer](https://www.llvm.org/docs/LibFuzzer.html), which performs the fuzzing
3. [qemu-user](https://qemu.readthedocs.io/en/latest/user/main.html), which provides user-space emulation.

TL/DR: check out our [libfuzzer-aarch64-crosscompile example](../../src/integration-tests/libfuzzer-aarch64-crosscompile/)

## Issues using qemu_user based LibFuzzer in OneFuzz

> There are a few notable limitations using qemu-user based libFuzzer targets on OneFuzz

Reviewer: ```suggestion
There are a few notable limitations when using `qemu-user` based libFuzzer targets in OneFuzz.
```
